src/physics/em/LivermoreParams.hh

```diff
@@ -0,0 +1,80 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file LivermoreParams.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "base/DeviceVector.hh"
+#include "io/ImportPhysicsVector.hh"
+#include "physics/material/Types.hh"
+#include "LivermoreParamsPointers.hh"
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Data management for the Livermore EPICS2014 Electron Photon Interaction
+ * Cross Section library.
+ */
+class LivermoreParams
+{
+  public:
+    //@{
+    //! Type aliases
+    using MevEnergy = units::MevEnergy;
+    //@}
+
+    struct SubshellInput
+    {
+        units::MevEnergy       binding_energy; //!< Ionization energy
+        std::vector<real_type> param_low;  //!< Low energy xs fit parameters
+        std::vector<real_type> param_high; //!< High energy xs fit parameters
+        std::vector<real_type> xs;         //!< Tabulated cross sections
+        std::vector<real_type> energy;     //!< Tabulated energies
+    };
+
+    struct ElementInput
+    {
+        ElementDefId        el_id;      //!< Index in MaterialParams elements
```

Comment:
The `el_id` here should be equal to the index of the `ElementInput` in the `Input::elements` vector, so this could probably be removed. To help with reading data, we might need to extend `MaterialParams` with some utility functions for interrogating the element IDs and Z numbers.

Reply:
Ahh ok, didn't realize that... then you're right, I can get rid of this.